packages/unit-testing/stubs/workbook/Chart.js

```diff
@@ -0,0 +1,163 @@
+define(['../dataset/Dataset', '../datasetLink/DatasetLink', './LimitingFilter', './ConditionalFilter', './Expression', './Legend', './Category', './Series'], function (
+	Dataset,
+	DatasetLink,
+	LimitingFilter,
+	ConditionalFilter,
+	Legend,
+	Category,
+	Series
+) {
+	/**
+	 * @class Chart
+	 * @classDescription Object representing a chart of a workbook
+	 * @constructor
+	 * @protected
+	 *
+	 * @since 2020.2
+	 */
+	function Chart() {
```

Comment:
Not in docs. No action needed.

Reply:
Same as with Category, e.g. it's used as return type of createChart function of the N/workbook